dbt_sugar/core/task/doc.py

```diff
@@ -26,7 +29,7 @@ class DocumentationTask(BaseTask):
 
     def __init__(self, flags: FlagParser, dbt_profile: DbtProfile) -> None:
         super().__init__()
-        self.columns_to_update: Dict[str, str] = {}
+        self.columns_to_update: Dict[str, Dict[str, Any]] = {}
```

Comment:
I'm a little confused where this gets filled in

Reply:
This is the response of the UI
{"COLUMN_NAME: {"description": "x", "tags" [], "tests": []}}